operators/cv2/imgcodecs/imdecode.hpp

```diff
@@ -33,26 +33,22 @@ struct KernelImageDecoder : BaseKernel {
     // Decode the image
     const std::vector<int32_t> encoded_image_sizes{1, static_cast<int32_t>(encoded_image_data_len)};
     const cv::Mat encoded_image(encoded_image_sizes, CV_8UC1,
-                            const_cast<void*>(static_cast<const void*>(encoded_image_data)));
+                                const_cast<void*>(static_cast<const void*>(encoded_image_data)));
```

Comment:
Suggest hiding whitespace diffs as clang-format has run when saving the edits causing lots of diffs. 
Other whitespace diffs are from conforming to 120 char limit but those might not get hidden.

<img width="242" alt="image" src="https://user-images.githubusercontent.com/979079/221051803-f8143b01-bc0f-4a42-86e5-dd2f62adb7f0.png">